src/main/java/the_warlord/powers/GashPower.java

```diff
@@ -0,0 +1,74 @@
+package the_warlord.powers;
+
+import basemod.interfaces.CloneablePowerInterface;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.MathUtils;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePatch;
+import com.evacipated.cardcrawl.modthespire.lib.SpirePrefixPatch;
+import com.megacrit.cardcrawl.actions.common.ApplyPowerAction;
+import com.megacrit.cardcrawl.actions.common.ReducePowerAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.PowerStrings;
+import com.megacrit.cardcrawl.monsters.AbstractMonster;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import com.megacrit.cardcrawl.powers.watcher.VigorPower;
+import the_warlord.WarlordMod;
+
+import java.util.Random;
+
+public class GashPower extends AbstractPower implements CloneablePowerInterface {
```

Comment:
The name we've been using most on Discord for this is Gush, rather than Gash. Both would make sense, but I think Gush is more specific to the bleeding out we're trying to suggest.

Also, the comments on BleedPower regarding CustomWarlordModPower also apply here.

Reply:
I'm fine with both Gush and Gash but it is true that personally I've used Gush all this time. So let's go with that.